pkg/sosreport/sosreport.jsx

```diff
@@ -477,7 +476,7 @@ const SOSBody = () => {
                 { title: <LabelGroup>{labels}</LabelGroup> },
                 {
                     title: <>{action}{menu}</>,
-                    props: { className: "pf-c-table__action table-row-action" }
+                    props: { className: "pf-v5-c-table__action table-row-action" }
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18806-20230516-073248-d32f52be-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)